src/banking_stage.rs

```diff
@@ -158,16 +146,15 @@ impl BankingStage {
         if !processed_transactions.is_empty() {
             let hash = Transaction::hash(&processed_transactions);
             // record and unlock will unlock all the successfull transactions
-            poh.record(hash, processed_transactions, working_bank)?;
+            poh.lock().unwrap().record(hash, processed_transactions)?;
         }
         Ok(())
     }
 
     pub fn process_and_record_transactions(
-        bank: &Bank,
+        bank: &Arc<Bank>,
```

Comment:
That looks weird. An artifact of a previous modification?

Reply:
Yea, I’ll fix tonight